src/CalendarBase.ts

```diff
@@ -1,10 +1,47 @@
 import { FORMAT } from './constants'
 import { parseDate } from './utils/time'
 
+export interface Options {
+  description?: string;
+  title?: string;
+  location?: string;
+  start: string;
+  end?: string;
+  recurrence?: Recurrence;
+}
+
+export interface Recurrence {
```

Comment:
for documentation purposes, we should document each property in this format:

```js
/**
 * Recurrence config options.
 */
interface IRecurrence {
  /** The Julian interval of how often this event should occur. */
  frequency: string
  /** The amount of time that will occur between recurrences. The scale of the time depends on {@link IRecurrence.frequency}. */
  interval: number
```

The descriptions of all of the types can be copied+pasted from [here](https://datebook.dev/docs/config.html#basic-config).